File: 9498/9498.rb
score = gets.chomp.to_i
grade = 'F'

case score
when 90..100 then grade = 'A'
when 80..89 then grade = 'B'
when 70..79 then grade = 'C'
when 60..69 then grade = 'D'
end

puts grade;

File: 9498/9498.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);
        
        int score = sc.nextInt();
        String grade = "F";
        
        if (score >= 90) {
            grade = "A";
        } else if (score >= 80) {
            grade = "B";
        } else if (score >= 70) {
            grade = "C";
        } else if (score >= 60) {
            grade = "D";
        }
        
        System.out.println(grade);
    }
}